pynio/block.py

```diff
@@ -118,3 +118,16 @@ def copy(self, name, instance=None):
         out._name = name
         out.config['name'] = name
         return out
+
+    def in_use(self, instance=None):
```

Comment:
The instance parameter complicates the Block by making it too powerful. This should be nice and simple. "Hey block, what services are you in?" It doesn't make sense to me to be able to ask a Block if it's in use in some Instance that's not associated with the block.